topics/vedtak.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  name: vedtak
```

Comment:
Burde vi kanskje vente med vedtak til vi har fått litt erfaring med kafka gjennom journalpost?

Reply:
egentlig enig, men det var et poeng for meg å lage et prosjekt som håndterte mer en en kø...